mico-core/src/main/java/io/github/ust/mico/core/web/ServiceController.java

```diff
@@ -358,6 +360,26 @@
         }
     }
 
+   @GetMapping("/{" + PATH_VARIABLE_SHORT_NAME + "}/{" + PATH_VARIABLE_VERSION + "}" + "/dependencyGraph")
```

Comment:
Isn't the basic idea of this PR and its relating issue, that we create an API endpoint to get all related `MicoService`s for a given `MicoApplication`?
Therefore, I would suggest to add an API endpoint in the `ApplicationController` and all related `MicoService`s and the edge list.

Reply:
We also want to display the dependency network of a single service so this endpoint is definitely needed for services. For applications it is also possible to use this endpoint even if it is a little bit more inefficient.